keyboards/lily58/keymaps/lily58l/keymap.c

```diff
@@ -0,0 +1,259 @@
+#include QMK_KEYBOARD_H
+
+#ifdef PROTOCOL_LUFA
+  #include "lufa.h"
+  #include "split_util.h"
+#endif
+#ifdef SSD1306OLED
+  #include "ssd1306.h"
+#endif
+
+#ifdef RGBLIGHT_ENABLE
+//Following line allows macro to read current RGB settings
+extern rgblight_config_t rgblight_config;
+#endif
+
+extern uint8_t is_master;
+
+#define _QWERTY 0
+#define _LOWER 1
+#define _RAISE 2
+#define _ADJUST 3
```

Comment:
```suggestion
enum layers {
    _QWERTY,
    _LOWER,
    _RAISE,
    _ADJUST,
};
```

Reply:
Thanks for the suggestion. C&P that from the [default keymap](https://github.com/qmk/qmk_firmware/blob/7d2cfadb550d931c5062dba78c7388af3d24964b/keyboards/lily58/keymaps/default/keymap.c#L18).
Looks cleaner 👍